mgmtfn/dockplugin/dockplugin.go

```diff
@@ -307,10 +319,15 @@ func createEndpoint(tenantName, hostname string) func(http.ResponseWriter, *http
 			return
 		}
 
-		// FIXME:
-		GetEndPointName(cereq.NetworkID, cereq.EndpointID)
+		// Get Service name
+		serviceName, err := GetEndPointName(cereq.NetworkID, cereq.EndpointID)
```

Comment:
Are you saying 'P' should not be capitalized in Endpoint?
I think it was capitalized by accident and then got cut and pasted..